### PR TITLE
XAMARINANDROID-3933-InlineApppointmentTextSize UG

### PR DESCRIPTION
**Pull request by MohanrajSF3559**
**Description:** InilineAppointmentTextSize and InlineAppointmentTimeTextSize UG content for Xamarin android calendar.

### CODE REVIEW

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 1]
```diff
@@ -120,6 +120,26 @@ Inline view mode
...
+
+ {% highlight c# %}
+  SfCalendar sfCalendar = new SfCalendar(this);
+            sfCalendar.MonthViewSettings.InlineAppointmentTextSize = 20;
+            sfCalendar.MonthViewSettings.InlineAppointmentTimeTextSize = 20;        
+            sfCalendar.ShowEventsInline = true;
+            sfCalendar.InlineViewMode = InlineViewMode.Inline;
+{% endhighlight %}
+
+ ![InlineAppointmentTextSize](images/Xamarin.Android-Calendar-InlineAppointmentTextSize.PNG)
```
Comment (Sasikumar3595) [inline xamarin-android/SfCalendar/Populating-Events.md:140]: Place the image on the above topic InlineAppointmentTextSize

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 2]
```diff
@@ -120,6 +120,26 @@ Inline view mode
...
+
+ ![InlineAppointmentTimeTextSize](images/Xamarin.Android-Calendar-InlineAppointmentTimeTextSize.PNG)
```
Comment (Sasikumar3595) [inline xamarin-android/SfCalendar/Populating-Events.md:142]: Check the images are on RTL

File: xamarin-android/SfCalendar/Populating-Events.md (hunk 1 above)
Comment (MohanrajSF3559) [reply xamarin-android/SfCalendar/Populating-Events.md]: I have changed the image and topic name. I have placed at needed place.

File: xamarin-android/SfCalendar/Populating-Events.md (hunk 2 above)
Comment (MohanrajSF3559) [reply xamarin-android/SfCalendar/Populating-Events.md]: I have changed image now.

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ N> The Inline function will be available only in MonthView with Single selection
 
 ## Adding events using Collection
 
-Calendar Events collection can be provided to `SfCalendar` using the following steps. [CalendarEventCollection](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.InlineEventToggledEventArgs.html#Com_Syncfusion_Calendar_InlineEventToggledEventArgs_CalendarEventCollection) is a class, which holds the details about the events to be rendered in `SfCalendar`. 
+Calendar Events collection can be provided to `SfCalendar` using the following steps. [CalendarEventCollection](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.InlineEventToggledEventArgs.html#Com_Syncfusion_Calendar_InlineEventToggledEventArgs_CalendarEventCollection) is a class, which holds the details about the events to be rendered in `SfCalendar`.
```
Comment (laksmipriyamurugesan) [inline xamarin-android/SfCalendar/Populating-Events.md:37]: Calendar Events collection can be provided to `SfCalendar` by following these steps. The [CalendarEventCollection](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.InlineEventToggledEventArgs.html#Com_Syncfusion_Calendar_InlineEventToggledEventArgs_CalendarEventCollection) is a class, which holds the details about the events to be rendered in `SfCalendar`.

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 4]
```diff
@@ -119,7 +119,51 @@ Inline view mode
 
 Agenda view mode
 
-![Calendar Inline Events](images/Xamarin.Android-Calendar-Agendaitemcustomview.png) 
+![Calendar Inline Events](images/Xamarin.Android-Calendar-Agendaitemcustomview.png)
+
+### InlineAppointmentTextSize
+
+Using `[InlineAppointmentTextSize](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html#Com_Syncfusion_Calendar_MonthViewSettings_InlineAppointmentTextSize)  property from the [MonthViewSettings](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html) class of SfCalendar control, you can customize the text size for the appointment details text in inline and agenda view.
```
Comment (laksmipriyamurugesan) [inline xamarin-android/SfCalendar/Populating-Events.md:126]: By using the `[InlineAppointmentTextSize](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html#Com_Syncfusion_Calendar_MonthViewSettings_InlineAppointmentTextSize) property from the [MonthViewSettings](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html) class of `SfCalendar` control, you can customize the text size for the appointment details text in inline and agenda view.

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 5]
```diff
@@ -119,7 +119,51 @@ Inline view mode
...
+            sfCalendar.MonthViewSettings.InlineAppointmentTextSize = 20;
+			sfCalendar.InlineViewMode = InlineViewMode.Agenda;
+            sfCalendar.ShowEventsInline = true;
+{% endhighlight %}
+
+![AgendaView with InlineAppointmentTextSize in Xamarin.Android Calendar](images/Xamarin.Android-Calendar-AgendaView-InlineAppointmentTextSize.JPG)
+
+ ### InlineAppointmentTimeTextSize
+
+Using `[InlineAppointmentTimeTextSize](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html#Com_Syncfusion_Calendar_MonthViewSettings_InlineAppointmentTimeTextSize) property from the [MonthViewSettings](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html) class of SfCalendar control, you can customize the time text size for the appointment details time in both inline and agenda view.
```
Comment (laksmipriyamurugesan) [inline xamarin-android/SfCalendar/Populating-Events.md:148]: By using the `[InlineAppointmentTimeTextSize](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html#Com_Syncfusion_Calendar_MonthViewSettings_InlineAppointmentTimeTextSize) property from the [MonthViewSettings](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html) class of `SfCalendar` control, you can customize the time text size for the appointment details time in both inline and agenda view.

**Review by Sasikumar3595 — APPROVED** [review]
This is fine.

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 6]
```diff
@@ -119,8 +119,53 @@ Inline view mode
 
 Agenda view mode
 
-![Calendar Inline Events](images/Xamarin.Android-Calendar-Agendaitemcustomview.png) 
-                                  
+![Calendar Inline Events](images/Xamarin.Android-Calendar-Agendaitemcustomview.png)
+
+### InlineAppointmentTextSize
+
+By using the [InlineAppointmentTextSize](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html#Com_Syncfusion_Calendar_MonthViewSettings_InlineAppointmentTextSize) property from the [MonthViewSettings](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html) class of SfCalendar` control, you can customize the text size for the appointment details text in inline and agenda view.
```
Comment (Nijamudeen-Mohamed) [inline xamarin-android/SfCalendar/Populating-Events.md:126]: Calendar`  - check syntax error

File: xamarin-android/SfCalendar/Populating-Events.md [hunk 7]
```diff
@@ -119,8 +119,53 @@ Inline view mode
...
+{% highlight c# %}
+  SfCalendar sfCalendar = new SfCalendar(this);
+            sfCalendar.MonthViewSettings.InlineAppointmentTextSize = 20;
+			sfCalendar.InlineViewMode = InlineViewMode.Agenda;
+            sfCalendar.ShowEventsInline = true;
+{% endhighlight %}
+
+ ### InlineAppointmentTimeTextSize
+
+By using the [InlineAppointmentTimeTextSize](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html#Com_Syncfusion_Calendar_MonthViewSettings_InlineAppointmentTimeTextSize) property from the [MonthViewSettings](https://help.syncfusion.com/cr/xamarin-android/Com.Syncfusion.Calendar.MonthViewSettings.html) class of SfCalendar` control, you can customize the time text size for the appointment details time in both inline and agenda view.
```
Comment (Nijamudeen-Mohamed) [inline xamarin-android/SfCalendar/Populating-Events.md:148]: Calendar`  - syntax error

File: xamarin-android/SfCalendar/Populating-Events.md (hunk 6 above)
Comment (MohanrajSF3559) [reply xamarin-android/SfCalendar/Populating-Events.md:126]: I have addressed the syntax error

File: xamarin-android/SfCalendar/Populating-Events.md (hunk 7 above)
Comment (MohanrajSF3559) [reply xamarin-android/SfCalendar/Populating-Events.md:152]: I have addressed the syntax error

File: xamarin-android/SfCalendar/Populating-Events.md (hunk 6 above)
Comment (MohanrajSF3559) [reply xamarin-android/SfCalendar/Populating-Events.md]: I have addressed the syntax error.

File: xamarin-android/SfCalendar/Populating-Events.md (hunk 7 above)
Comment (MohanrajSF3559) [reply xamarin-android/SfCalendar/Populating-Events.md]: I have addressed the syntax error.

**Review by Nijamudeen-Mohamed — APPROVED** [review]

**Pull request merged**